bokehjs/src/coffee/common/hittest.coffee

```diff
@@ -35,7 +35,13 @@ class HitTestResult
       indices: []
     }
     # 2d for all for multilines and multi patches
-    @['2d'] = {indices: []}
+    @['2d'] = {
+      # index of the glyph in the multiglyph
+      indices: []
+      # object containing glyph_index: [point_index] of the nearest
+      # point to the crossed segment
+      point_indices: {}
+    }
```

Comment:
>
1d indices would say "which line"
2d indices would say "which segment" on the lines

The hittesting module didn't seem written with that pattern in mind (it was performing lookups for `results['2d'].indices`, but I see thats how hit-testing for patches was implemented. So I changed the multiline hit-testing to match and cleaned up some old code.

I like having the '2d' value be an object better anyway.